package.json

```diff
@@ -57,8 +58,10 @@
     "lerna": "^3.4.3",
     "pre-commit": "^1.2.2",
     "prettier": "^1.15.1",
-    "typescript": "^3.0.3",
-    "typescript-eslint-parser": "^18.0.0"
+    "typescript": "^3.2.2",
+    "typescript-eslint-parser": "^18.0.0",
+    "webpack": "next",
```

Comment:
why webpack should be on root?

Reply:
I might have misunderstood, correct me if i'm wrong. Since webpack is being used by server and web, makes sense to have it on the root other than on each package, right?

Reply:
Is this used by all packages as well?

Reply:
Have to double check, but I guess not.

Reply:
I prefer to have the package only in packages that consume them

yarn will hoist them if they match the same version anyway